core/metrics_manager.js

```diff
@@ -10,31 +10,35 @@
  */
 'use strict';
 
-goog.provide('Blockly.FlyoutMetricsManager');
-goog.provide('Blockly.MetricsManager');
-
-goog.requireType('Blockly.IMetricsManager');
-goog.require('Blockly.registry');
-goog.require('Blockly.utils.Size');
-goog.require('Blockly.utils.toolbox');
-
-goog.requireType('Blockly.IFlyout');
-goog.requireType('Blockly.IToolbox');
-goog.requireType('Blockly.utils.Metrics');
-goog.requireType('Blockly.WorkspaceSvg');
+goog.module('Blockly.MetricsManager');
+goog.module.declareLegacyNamespace();
+
+/* eslint-disable-next-line no-unused-vars */
+const IFlyout = goog.requireType('Blockly.IFlyout');
+/* eslint-disable-next-line no-unused-vars */
+const IMetricsManager = goog.requireType('Blockly.IMetricsManager');
+/* eslint-disable-next-line no-unused-vars */
+const IToolbox = goog.requireType('Blockly.IToolbox');
+/* eslint-disable-next-line no-unused-vars */
+const Metrics = goog.requireType('Blockly.utils.Metrics');
+const Size = goog.require('Blockly.utils.Size');
+/* eslint-disable-next-line no-unused-vars */
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const registry = goog.require('Blockly.registry');
+const toolbox = goog.require('Blockly.utils.toolbox');
```

Comment:
I think this should probably be renamed to `toolboxUtils` or something that is more clear that this is not the actual toolbox.